src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/BedAssignmentHistory.java

```diff
@@ -45,6 +45,18 @@
         )
     }
 )
+@NamedEntityGraph(
+    name = "bed-history-with-location",
+    attributeNodes = @NamedAttributeNode(value = "location", subgraph = "bed-location"),
+    subgraphs = {
+        @NamedSubgraph(
+            name = "bed-location",
+            attributeNodes = {
+                @NamedAttributeNode("livingUnit")
```

Comment:
Is livingUnit going to be added to AgencyInternalLocation too? (sorry probably a different ticket 😄 )